bot.py

```diff
@@ -22,7 +23,7 @@ def main():
         logger.error("Cannot find bot_token env. variable that is required. Stopping.")
         exit(-1)
 
-    updater = Updater(os.environ["bot_token"])
+    updater: Updater = Updater(os.environ["bot_token"])
```

Comment:
what kind of syntax it is?

Reply:
Python type hints -> https://docs.python.org/3/library/typing.html